platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -323,14 +352,76 @@ - (MGLImage *)blurredAttributionBackground:(CIImage *)backgroundImage
     CGImageRef cgimg = [ctx createCGImage:blurredImage fromRect:[backgroundImage extent]];
     
 #if TARGET_OS_IPHONE
-    
     return [UIImage imageWithCGImage:cgimg];
 #else
-    
     return [[NSImage alloc] initWithCGImage:cgimg size:[backgroundImage extent].size];
 #endif
 }
 
+- (MGLSnapshotAttributionOptions *)attributionOptionsForSize:(CGSize)snapshotSize attributionInfo:(NSArray *)attributionInfo
+{
+    NSMutableArray *attributionOptions = [NSMutableArray array];
```

Comment:
It’s a bit confusing that an array of attribution options is called `attributionOptions`, as though it were only a single MGLSnapshotAttributionOptions object.